### PR TITLE
Use promises and ES2015ify tests

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,10 +27,17 @@
...
+    "pinkie-promise": "^1.0.0",
     "tasklist": "^1.0.0"
   },
   "devDependencies": {
-    "ava": "0.0.4"
+    "ava": "*",
+    "xo": "*"
+  },
+  "xo": {
+    "ignore": [
```
Comment (SamVerschueren) [inline package.json]: [`ignores`](https://github.com/sindresorhus/xo#ignores) instead of `ignore`.

File: index.js [hunk 2]
```diff
@@ -2,51 +2,34 @@
...
+	opts = opts || {};
 
-	tasklist(function (err, data) {
-		if (err) {
-			cb(err);
-			return;
-		}
-
-		var ret = data.map(function (el) {
+	return pify(tasklist, Promise)().then(function (data) {
```
Comment (sindresorhus) [inline index.js]: Can you update to use `tasklist` 2.0.0?

File: readme.md [hunk 3]
```diff
@@ -26,7 +26,9 @@ psList(function (err, data) {
 
 ## API
 
-### psList([options], callback)
+### psList([options])
+
+Returns a promise that resolves to an array containing the running processes.
```
Comment (sindresorhus) [inline readme.md]: > Returns a promise for an array containing the running processes.

File: test.js [hunk 4]
```diff
@@ -1,23 +1,14 @@
...
-		}));
-	});
+import test from 'ava';
+import fn from './';
+
+test(async t => {
+	const binName = process.platform === 'win32' ? 'node.exe' : 'ava';
+	const list = await fn();
+
+	t.true(list.some(x => x.name.indexOf(binName) !== -1));
```
Comment (sindresorhus) [inline test.js]: Array#includes()

Comment (kevva) [inline test.js]: How do you check for a value in an object?

Comment (sindresorhus) [inline test.js]: That should have been `String#includes()`.

**Pull request merged**